packages/client-core/src/admin/components/resources/ResourceTable.tsx

```diff
@@ -79,22 +77,6 @@ export default function ResourceTable({ search }: { search: string }) {
           >
             <HiEye className="place-self-center text-theme-primary" />
           </Button>
-          <Button
-            rounded="full"
-            className="h-8 w-8 justify-center border border-theme-primary bg-transparent p-0"
-            onClick={() => {
-              PopoverState.showPopupover(
-                <ConfirmDialog
-                  text={`${t('admin:components.resources.confirmResourceDelete')} '${el.key}'?`}
-                  onSubmit={async () => {
-                    await Engine.instance.api.service(staticResourcePath).remove(el.id)
-                  }}
-                />
-              )
-            }}
-          >
-            <HiTrash className="place-self-center text-theme-iconRed" />
-          </Button>
```

Comment:
why are we removing the ability to delete in admin?

Reply:
oh whoops, i need to add this back.